pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1118,13 +1118,26 @@ func maybeRetryBoot(ctx *domainContext, status *types.DomainStatus) {
 	}
 	defer file.Close()
 
+	if err := hyper.Task(status).VirtualTPMSetup(status.DomainName, agentName, ctx.ps, warningTime, errorTime); err != nil {
+		log.Errorf("Failed to setup virtual TPM for %s: %s", status.DomainName, err)
+		config.VmConfig.VirtualTPM = false
```

Comment:
Mmm... I always consider config to be "read-only" in the component to which it comes. 
If I need to save some info about the domain, I need to create a corresponding field in DomainStatus. DomainStatus is under the DomainManager control. Usually, when you update a field, you have to publish the status to make the change visible.

Reply:
changed.